README.md

```diff
@@ -81,26 +90,29 @@ To extract metrics data from the SDK import the following into your content view
+| `sdk.strictPulseRate`        | (Double)                                   | The strict pulse rate (high confidence average over spot duration)     |
+| `sdk.strictBreathingRate`    | (Double)                                   | The strict breathing rate (high confidence average over spot duration) |
+| `sdk.pulseValues`            | [(time: Double, value: Double)]            | Pulse rates                                                            |
+| `sdk.pulseConfidence`        | [(time: Double, value: Double)]            | Pulse rate confidences                                                 |
+| `sdk.pulsePleth`             | [(time: Double, value: Double)]            | Pulse waveform or pleth                                                |
+| `sdk.hrv`                    | [(time: Double, value: Double)]            | Pulse rate variability (RMSSD) **(Requires 60+ second Spot Duration)** |
+| `sdk.breathingValues`        | [(time: Double, value: Double)]            | Breathing rates                                                        |
+| `sdk.breathingPleth`         | [(time: Double, value: Double)]            | Breathing movement waveform or pleth                                   |
+| `sdk.breathingAmplitude`     | [(time: Double, value: Double)]            | Breathing rate confidences                                             |
+| `sdk.apnea`                  | [(time: Double, value: Bool)]            | Apnea detection                                                        |
+| `sdk.breathingBaseline`      | [(time: Double, value: Double)]            | Breathing baseline                                                     |
+| `sdk.phasic`                 | [(time: Double, value: Double)]            | Phasic (ie changes in relative blood pressure)                         |
+| `sdk.rrl`                    | [(time: Double, value: Double)]            | Respiratory line length                                                |
+| `sdk.ie`                     | [(time: Double, value: Double)]            | The inhale exhale ratio                                                |
+| `sdk.uploadDate`             | (String)                                   | upload date time                                                       |
+| `sdk.version`                | (String)                                   | the version of API used                                                |
+| `sdk.userID`                 | (String)                                   | the user ID                                                            |
+| `sdk.jsonMetrics`            | (Dictionary)                               | containing the metrics available according to your api key. See Data Format below for the contents and structure. **Warning: json structure is subject to change use at your own risk.** |
 
 ### Data Format
-`sdk.jsonMetrics` is structured as follows:
+`sdk.jsonMetrics` is structured as follows: 
 ```json
```

Comment:
@1-ashraful-islam can you delete the rr_spec and hr_spec from the json structure? We are not returning that anymore.

Reply:
@1-ashraful-islam we don't even actually have the json extraction on android anymore. I would maybe just remove json extraction

Reply:
both of these have been addressed.